src/constants.ts

```diff
@@ -113,14 +113,19 @@ export const ABSOLUTE_PATH = [{ value: ABSOLUTE_TOKEN, rank: 0 }] as SimplePath
 export const ALLOW_SINGLE_CONTEXT = false
 
 export const FIREBASE_CONFIG = {
-  apiKey: 'AIzaSyB7sj38woH-oJ7hcSwpq0lB7hUteyZMxNo',
-  authDomain: 'em-proto.firebaseapp.com',
-  databaseURL: 'https://em-proto.firebaseio.com',
-  projectId: 'em-proto',
-  storageBucket: 'em-proto.appspot.com',
+  apiKey: 'AIzaSyBpC2KffoQNufewJ4uS-WS72sqeXVJ5T-U',
+  authDomain: 'em-search-test.firebaseapp.com',
+  databaseURL: 'https://em-search-test-default-rtdb.firebaseio.com/',
+  projectId: 'em-search-test',
+  storageBucket: 'gs://em-search-test.appspot.com/',
   messagingSenderId: '91947960488'
 }
```

Comment:
How about we define `FIREBASE_CONFIG` in its own file and gitignore it, e.g. `firebase-config.js`? This will allow us to use different environments without affecting the production configuration. We can include `firebase-config.sample.js` in the repo to help developers.

The variables could also go in environment variables, not sure which is better here.

Reply:
I was thinking about that. I will make the changes.